eclipse-scout-core/src/util/strings.ts

```diff
@@ -379,20 +379,36 @@ export const strings = {
     return strings.equals(a, b, true);
   },
 
+  /**
+   * Adds the given prefix to the start of the given string and returns the result.
+   * If either of the given arguments is null or empty, the original string is returned unchanged, i.e. without prefix.
+   */
+  addPrefix(string: string, prefix: string): string {
+    return string && prefix ? prefix + string : string;
```

Comment:
why do we not use _box_ here?

Reply:
Because StringUtility#box returns `""` when the original string is `null`. This is somewhat useful when the result is directly concatenated to another string. But otherwise, the "is-null" information about the original string is lost.

I think these methods more accurately represent the operation we want to perform. `box` should only be used when a string should be surrounded by two other strings (usually some kind of brackets), i.e. a combination of `addPrefix` and `addSuffix`. Adding a string only on one end by adding "nothing" on the other side feels unnecessarily complicated.

Also, because of the symmetry with `removePrefix`/`removeSufffix`.